modules/client/examples.md

```diff
@@ -2350,3 +2350,297 @@ console.log(members);
+  actions: [],
+};
+
+const steps = client.methods.executeProposal(proposalParams);
+for await (const step of steps) {
+  try {
+    switch (step.key) {
+      case ExecuteProposalStep.EXECUTING:
+        console.log(step.txHash);
+        break;
+      case ExecuteProposalStep.DONE:
+        break;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
+```
+
+### Loading the a proposal by proposalId (admin plugin)
+
+```ts
+import {
+  AdminProposal,
+  ClientAdmin,
+  Context,
+  ContextPlugin,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an ERC20 client
+const client = new ClientAdmin(contextPlugin);
+
+const proposalId = "0x12345...";
+
+const proposal: AdminProposal | null = await client.methods.getProposal(
+  proposalId,
+);
+console.log(proposal);
+/*
+{
+  id: "0x12345...",
+  dao: {
+    address: "0x1234567890123456789012345678901234567890",
+    name: "Cool DAO"
+  };
+  creatorAddress: "0x1234567890123456789012345678901234567890",
+  metadata: {
+    title: "Test Proposal",
+    summary: "test proposal summary",
+    description: "this is a long description",
+    resources: [
+      {
+        url: "https://dicord.com/...",
+        name: "Discord"
+      },
+      {
+        url: "https://docs.com/...",
+        name: "Document"
+      }
+    ],
+    media: {
+      header: "https://.../image.jpeg",
+      logo: "https://.../image.jpeg"
+    }
+  };
+  creationDate: <Date>,
+  actions: [
+    {
+      to: "0x12345..."
+      value: 10n
+      data: [12,13,154...]
+    }
+  ],
+  status: "Executed",
+  administratorAddress: "0x1234567890123456789012345678901234567890"
+  pluginAddress: "0x1234567890123456789012345678901234567890"
+  proposalId: 0n
+}
+*/
+```
+
+### Loading the list of proposals (address list plugin)
+
+```ts
+import {
+  AdminProposalListItem,
+  ClientAdmin,
+  Context,
+  ContextPlugin,
+  IProposalQueryParams,
+  ProposalSortBy,
+  ProposalStatus,
+  SortDirection,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an address list client
+const client = new ClientAdmin(contextPlugin);
+
+const queryParams: IProposalQueryParams = {
+  skip: 0, // optional
+  limit: 10, // optional,
+  direction: SortDirection.ASC, // optional
+  sortBy: ProposalSortBy.POPULARITY, //optional
+  status: ProposalStatus.ACTIVE, // optional
+};
+
+const proposals: AdminProposalListItem[] = await client.methods
+  .getProposals(queryParams);
+console.log(proposals);
+/*
+[
+  {
+    id: "0x12345...",
```

Comment:
I know, but the subgraph is using `pluginAdddress_proposalId` as `id` because it has to be unique, the bigint proposal ID is in the field `proposalId`